aptos-move/framework/aptos-token-objects/sources/collection.move

```diff
@@ -440,6 +440,27 @@ module aptos_token_objects::collection {
         }
     }
 
+    spec increment_supply {
+        pragma aborts_if_is_partial;
```

Comment:
What aspect is not specified and making this spec to be partial?

Reply:
I only specified the part where the aggregator_v2 is used. Other parts are not specified.